index.js

```diff
@@ -1,4 +1,12 @@
-'use strict';
-// we're gonna use strict  mode in all
+try {
+    const OpenBrowser = require('./cfeScraping/index.js');
```

Comment:
recommend using another name for the import, as you see is a little redundant OpenBrowser.OpenBrowser();